exercises/K-functions-parameters/exercise2.js

```diff
@@ -1,5 +1,8 @@
 // Declare your function first
+function divide (a, b) {
+return a / b ;
```

Comment:
Indentation error! This line should be indented as it is inside a code block.

Reply:
Okay. I fixed it. :+1: